### ISSUE TITLE
Create on/off toggle for Google Docs

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#220]
This story was incorporated into #235. There is no separate PR for this card. Keeping this card for QA purposes. 
++++++++++++++++++++++++++++++++++++++++++++++++++++

As a GT user, I would like to be able to turn the google doc "whiteboard" off or on, so that I can use Riff to best accommodate the needs of my particular meeting. 

As a PM, I would like the Google doc to be on by default, so that users know it's a feature, and do not have to find it and turn it on to use it. 

** Acceptance Criteria **
- [ ] Add two new document icons (in comments below) between the mic and screen share options, so that the user has a control to open and close the doc. 
- [x] When the document is closed, the vertical video tile strip should expand (tile) to use the full width of the screen as it does in a regular Riff Video display, so that users can see each other better. 
- [x] When a document is opened, the tiles should revert back to the vertical display. 
- [ ] Have the document open as the default behavior when a user joins a room, so that the feature is evident to the user. 
 
*** With screen sharing ***
- [x] Enable screen sharing in the GT build if it is not already. 
- [x] When someone in the meeting screen shares, close the google doc (if displayed) for all users, so that the document real estate can be used for the screen sharing display. Keep users vertically tiled.
- [ ] When screen sharing is ended, the google doc should return to the state it was in (either open or closed) for each user.

**Comment by jaedoucette** [comment]
We think that, in the event google docs cannot be made for some reason (or are taking a while to make), this button should simply be grayed out, or a "loading"-type icon should be displayed, maybe with a tooltip that informs them about the delay.

**Comment by adonahue** [comment]
I will add that functionality to the story.

**Comment by adonahue** [comment]
![iconmonstr-note-14.svg](https://images.zenhubusercontent.com/5cbf1c836caba0089f532d44/999800b3-b4c5-44ee-8629-b6d57c32ac1c)

![iconmonstr-note-32.svg](https://images.zenhubusercontent.com/5cbf1c836caba0089f532d44/0f97cd3e-3341-4b7d-9113-41b08a00fa68)

**Comment by adonahue** [comment]
@jordanreedie @mlippert  - what is the status of this story? I think everything here (except the icons) are deployed on staging. Is this still waiting on a code review?

**Comment by adonahue** [comment]
Testing Notes:

* New doc icons lower-priority
* On join, document state is the same as the state on leave, not always on by default. 
* On stop share view always opens to the doc - not the previous state of the doc (either open or closed). Looking at a note in #235 which says this was intentional as a simpler implementation. 

Accepting story as-is. Verified in Chrome.
